File: app/src/main/java/zmuzik/slidingpuzzle2/gamescreen/GameScreenView.java
package zmuzik.slidingpuzzle2.gamescreen;

/**
 * Created by Zbynek Muzik on 2017-04-03.
 */

public interface GameScreenView {

    void finishWithMessage(int stringId);

    void loadPicture(String pictureUri);

    int getMaxScreenDim();

    void hideShuffleIcon();
}

File: app/src/main/java/zmuzik/slidingpuzzle2/gamescreen/GameScreenView.kt
package zmuzik.slidingpuzzle2.gamescreen

/**
 * Created by Zbynek Muzik on 2017-04-03.
 */

interface GameScreenView {

    fun finishWithMessage(stringId: Int)

    fun loadPicture(pictureUri: String)

    fun getMaxScreenDim(): Int

    fun hideShuffleIcon()
}
